### PR TITLE
docs(clustered): add section for troubleshooting licensing

### PR DESCRIPTION
**Pull request by jdockerty**
As licensing is moving to being utilised by customers, we'll need some documentation on how to troubleshoot/verify that things are working as intended.

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/clustered/admin/licensing.md [hunk 1]
```diff
@@ -85,21 +87,22 @@ per hour) while running.
...
-Once a valid `License` resource is applied, new pods will begin to start up normally.
-Licenses are validated when the `License` resource is applied. If the license
-is invalid when you attempt to apply it, the InfluxDB clustered license
-controller will not add or update the required secret.
+InfluxDB validates a license when you apply it.
+If the license is invalid when you try to apply it, the `license controller`
+won't add or update the required secret.
 
-## Renew your license
+## Renew your licensejk
```
Comment (sanderson) [inline content/influxdb/clustered/admin/licensing.md:102]:
```suggestion
## Renew your license
```

**Review by jstirnaman — COMMENTED** [review]
@jdockerty I just made revisions in the branch. ~I'll add you to that PR and close this one.~

**Review by sanderson — APPROVED** [review]
Just one typo fix and I think it's good to go.

**Comment by jstirnaman** [comment]
> @jdockerty I just made revisions in the branch. ~I'll add you to that PR and close this one.~

Can you review my changes?

**Comment by jdockerty** [comment]
@jstirnaman / @sanderson  Looks great, thanks both as always!

**Pull request merged**